test/index.js

```diff
@@ -636,4 +694,80 @@ describe('KeyringController', function () {
       expect(keyringController.memStore.getState().encryptionKey).toBeNull();
     });
   });
+
+  describe('exportAccount', function () {
+    it('returns the private key for the public key it is passed', async () => {
+      await keyringController.createNewVaultAndRestore(
+        password,
+        walletOneSeedWords,
+      );
+      const privateKey = await keyringController.exportAccount(
+        walletOneAddresses[0],
+      );
+      expect(privateKey).toStrictEqual(walletOnePrivateAddress[0]);
+    });
+  });
+
+  describe('signing methods', function () {
+    beforeEach(async () => {
+      await keyringController.createNewVaultAndRestore(
+        password,
+        walletOneSeedWords,
+      );
+    });
+
+    it('signMessage', async () => {
+      const inputParams = {
+        from: walletOneAddresses[0],
+        data: '0x879a053d4800c6354e76c7985a865d2922c82fb5b3f4577b2fe08b998954f2e0',
+        origin: 'https://metamask.github.io',
+      };
+      const result = await keyringController.signMessage(inputParams);
+      expect(result).toBe(
```

Comment:
Nit: for values like this that a reader can't reasonably be expected to be able to read, it might be worth using an inline snapshot. This communicates to the reader that the value was auto-generated, and makes it easier to update if we do ever intend to change it.

Reply:
done [here](https://github.com/MetaMask/KeyringController/pull/140/commits/3b35dceaf4c0a1170f516d19b41d7f5f8bd57c93)